README.md

```diff
@@ -52,7 +48,7 @@ This will initiate the model training and inference action.
 
 2. **Inference Mode** : 
 
-Similar to the `train-ttm.yaml` file, you can add another file called `predict-ttm.yaml` file to `.github/worklows/` like [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.github/workflows/predict-ttm.yaml). This file has `MODE` as `0` which would enable just inference on the new incoming Pull Request and add a comment to the pull request specifying the approximate time it will take to be merged.
+Similar to the `train-ttm.yaml` file, you can add another file called `predict-ttm.yaml` file to `.github/worklows/` folder in your repository like [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.github/workflows/predict-ttm.yaml). This file has `MODE` as `0` which would enable just inference on the new incoming pull requests and add a comment on the pull request specifying the approximate time it will take to be merged.
```

Comment:
```suggestion
Similar to the `train-ttm.yaml` file, you need to add another file called `predict-ttm.yaml` to the `.github/worklows/` folder in your repository that looks like [this](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/.github/workflows/predict-ttm.yaml). This file has set `MODE` to `0` which will enable inference on all new incoming pull requests and add a comment on the pull request specifying the approximate time it will take to be merged.
```